examples/DataApi-NFT/template/modules/shared_data/src/main/scala/com/my/nft_example/shared_data/Utils.scala

```diff
@@ -0,0 +1,48 @@
+package com.my.nft_example.shared_data
+
+import com.my.nft_example.shared_data.Data.{NFTUpdate, State}
+import io.circe.parser
+import io.circe.syntax.EncoderOps
+
+import java.net.{MalformedURLException, URISyntaxException, URL}
+import java.nio.charset.StandardCharsets
+
+object Utils {
+
+  @throws[MalformedURLException]
+  @throws[URISyntaxException]
+  def isValidURL(url: String): Boolean = try {
+    new URL(url).toURI
+    true
+  } catch {
+    case _: MalformedURLException =>
+      false
+    case _: URISyntaxException =>
+      false
+  }
+
+  def customUpdateSerialization(update: NFTUpdate): Array[Byte] = {
+    println("Serialize UPDATE event received")
+    println(update.asJson.deepDropNullValues.noSpaces)
+    update.asJson.deepDropNullValues.noSpaces.getBytes(StandardCharsets.UTF_8)
```

Comment:
Not sure how many times this and logic below are repeated? Maybe duplication

Reply:
I don't think so, it's used only on the serializations